opentreemap/treemap/css/sass/partials/pages/_admin.scss

```diff
@@ -256,7 +257,8 @@
             border-bottom: 1px solid #EEE;
 
             &.active {
-                background-color: #F7F3DF;
+                background-color: $warning-color;
+                background-color: rgba($warning-color, .75);
```

Comment:
Did you intend to include `background-color` here twice? An explanatory comment might be helpful if this is intended.